autocomplete_mention/main.js

```diff
@@ -0,0 +1,89 @@
+/*global $:false */
```

Comment:
Simply add jquery to the eslintrc file
http://eslint.org/docs/user-guide/configuring.html#specifying-environments